lib/hackney/income/tenancy_prioritiser/tenancy_classification.rb

```diff
@@ -36,10 +37,22 @@ def validate_wanted_action(wanted_action)
           raise ArgumentError, "Tried to classify a case as #{wanted_action}, but this is not on the list of valid classifications."
         end
 
+        def send_informal_agreement_breach_letter?
+          return false if @criteria.number_of_broken_agreements < 1
```

Comment:
Can it be a negative? If not: 

```suggestion
          return false if @criteria.number_of_broken_agreements.zero?
```